lunes_cms/static/js/toggle_document_form_fields.js

```diff
@@ -22,76 +22,79 @@ $(document).ready(() => {
 
     // Show alert message
-    messageBox = document.createElement("div");
-    message = document.createTextNode(data["message"]);
-    messageBox.append(message);
-    result.append(messageBox);
+    addDetailInfo(result, "message", "normal");
+
     // Show the duplicated word with its word type
-    wordBox = document.createElement("div");
-    word = document.createTextNode(data["word"]);
-    wordBox.style.fontWeight = "bold";
-    wordBox.append(word);
-    result.append(wordBox);
+    addDetailInfo(result, "word", "bold");
+
     // Show its definition too for more detail
-    definitionBox = document.createElement("div");
-    definition = document.createTextNode(data["definition"]);
-    definitionBox.append(definition);
-    result.append(definitionBox);
+    addDetailInfo(result, "definition", "normal");
+
     // Show related training sets
-    trainingSetsBox = document.createElement("div");
-    trainingSets = document.createTextNode(data["training_sets"]);
-    trainingSetsBox.append(trainingSets);
-    result.append(trainingSetsBox);
+    addDetailInfo(result, "training_sets", "normal");
   } else {
     // If there is no duplicate, use green background
-    result.style.backgroundColor = "#72f399";
-    result.style.padding = "25px"
+    result = createSearchResultField("#72f399");
+
     // Show message that no duplicate was found
-    messageBox = document.createElement("div");
-    message = document.createTextNode(data["message"]);
-    messageBox.append(message);
-    result.append(messageBox);
+    addDetailInfo(result, "message", "normal");
   }
   
   parent.prepend(result);
 }
 
+// Create a zone in which result duplicate search will be shown
+function createSearchResultField(backgroundColor){
+  messageField = document.createElement("div");
+  messageField.setAttribute("id", "result");
+  messageField.style.padding = "25px";
+  messageField.style.backgroundColor = backgroundColor;
 
-$(document).ready(() => {
-  $("#id_word").change((event) => {
-    if ($(event.target).val().length > 0) {
-      $.ajax({
-        type: 'GET',
-        url: '/api/search_duplicate/' + $(event.target).val(),
-        dataType: "json",
-        success: function(data) {
-            showDuplicates(data, $(event.target).closest(".card-body"));
-        }
-      })
-    } else {
-      removeDuplicationCheckMessage();
-    }
-  });
-  $("#id_word_type").trigger("change");
-});
\ No newline at end of file
+  return result;
+}
+
+// Create and add detail information
+function addDetailInfo(resultField, infoType, fontWeight) {
+  detailInfoField = document.createElement("div");
+  detailedInfoField.style.fontWeight = fontWeight;
+  detailInfo = document.createTextNode(data[infoType]);
+  detailInfoField.append(detailInfo);
+  resultField.append(detailInfoField);
+}
+
+// Remove the duplicate search result
+function removeDuplicationCheckMessage(){
+  var existingMessage = document.getElementById("result");
```

Comment:
```suggestion
  const existingMessage = document.getElementById("result");
```
`var` shouldn't be used in modern Javascript code anymore. Instead `const` for non-changeable or `let` for changeable variables should be used instead. More on this [here](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/) :)